cg/services/order_validation_service/models/discriminators.py

```diff
@@ -0,0 +1,7 @@
+from typing import Any
+
+
+def has_internal_id(v: Any) -> str:
```

Comment:
Should this not return a bool?

Reply:
It needs to return a matching "Tag". But maybe the naming should reflect that? `get_tag` or something?

Reply:
Sounds good!

Reply:
Resolved in Slack - keeping naming for now since it gives more info on what decides what model to parse